examples/models/object_detection/retina_net/dummy_dataset_visualization.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras import optimizers
+
+import keras_cv
+from keras_cv import bounding_box
+
+
+def main():
+    bounding_box_format = "xywh"
+    retina_net = keras_cv.models.RetinaNet(
+        classes=1,
+        bounding_box_format=bounding_box_format,
+        backbone="resnet50",
+        backbone_weights=None,
+        include_rescaling=False,
+    )
+    loss = keras_cv.losses.ObjectDetectionLoss(
+        classes=1,
+        classification_loss=keras_cv.losses.FocalLoss(
+            from_logits=True, reduction="none"
+        ),
+        box_loss=keras_cv.losses.SmoothL1Loss(l1_cutoff=1.0, reduction="none"),
+        reduction="sum",
+    )
+
+    retina_net.compile(
+        optimizer=optimizers.Adam(),
+        loss=loss,
+        metrics=[
+            keras_cv.metrics.COCOMeanAveragePrecision(
+                class_ids=range(1),
+                bounding_box_format=bounding_box_format,
+                name="MaP",
+            ),
+            keras_cv.metrics.COCORecall(
+                class_ids=range(1),
+                bounding_box_format=bounding_box_format,
+                name="Recall",
+            ),
+        ],
+    )
+
+    xs, ys = _create_bounding_box_dataset(bounding_box_format)
+
+    while True:
```

Comment:
Rather than `while True: model.fit(epochs=10)`, this should be a callback that runs the same code in `on_epoch_end` every 10 epochs

Reply:
Good point, I'll refactor this.  I mainly did this just so that I could prove to myself the model was learning at one point - but I'll rewrite this to be a better example.